### ISSUE TITLE
Trig carousel inside translation string

### ISSUE THREAD

**Issue by peterwesterlund** [angular-translate/angular-translate#1685]
How do I trig the angular carousel been loaded along with a translation string?

**Comment by tspaeth** [comment]
Please ask "how do I..." questions on Stackoverflow: http://stackoverflow.com/questions/tagged/angular-translate - you'll reach a bigger audience. 

This issue tracker is mainly the feature-/bugtracker for angular-translate. But you're always free to add a feature request later on. 

Thanks.

**Comment by peterwesterlund** [comment]
Well, as you can see on the link you referred to, I have already asked the question there but without any response. And I have been working with this the whole day. Trying other carousels and try learning how to do on my own. But aren't getting any further. This was kind of my last hope.

**Comment by tspaeth** [comment]
Honestly, the question is 7 hours old - are you really expecting answers that fast?

**Comment by peterwesterlund** [comment]
Yes, the answers usually comes after just minutes or at most some hour. Longer time tend to reduce the hope for answers. But of course I still hope for a response there. There's still hope. But I honestly thought I were allowed to ask the question here. Didn't meant to do anything forbidden. I just waited to write here because I expected more quickly response on Stackoverflow. I have actually struggled with this problem for some days now. So writing on Stackoverflow wasn't my first thing to do either. Always trying hard to resolve my issues alone before asking for help.

When nothing worked, it also made me think it might be a bug. What do I know? Since I can't find any descriptions about how to do this in the documentation. Maybe it **should** work without any extra coding.
